src/main/scala/nl/knaw/dans/easy/multideposit/actions/CreateOutputDepositDir.scala

```diff
@@ -28,12 +28,28 @@ case class CreateOutputDepositDir(row: Int, datasetID: DatasetID)(implicit setti
   private val dirs = depositDir :: bagDir :: metadataDir :: Nil
 
   override def checkPreconditions: Try[Unit] = {
+    for {
+      _ <- checkDatasetIdIsValid
+      _ <- checkDirectoriesDoNotExist
```

Comment:
Maybe Joke refers to the files that are mentioned but do not exist; see checkPreconditions of AddFileMetadataToDeposit ?